mediator/src/main/scala/org/hyperledger/identus/mediator/DIDCommRoutes.scala

```diff
@@ -50,10 +51,14 @@ object DIDCommRoutes {
               def transmissionType = Transport.TransmissionType.SingleTransmission
               def id: TransportID = TransportID.http(req.headers.get("request_id"))
               def inbound: ZStream[Any, Transport.InErr, SignedMessage | EncryptedMessage] =
-                ZStream.fromQueue(inboundQueue)
+                ZStream
+                  .fromQueue(inboundQueue)
+                  .timeout(10.seconds)
```

Comment:
I don't think having a timeout here is good.
Because in the case of web-sockets you may want to keep it open for a longer time.

the `.ensuring(inboundQueue.shutdown)` LGTM